PWGCF/Flow/Tasks/flowPbpbPikp.cxx

```diff
@@ -174,8 +177,18 @@ struct FlowPbpbPikp {
     useTVXinTRD
   };
 
-  std::vector<int> fintEvCuts = cfgUseEventCuts;
-  std::vector<bool> eventCuts;
+  enum TrackCutTypes {
+    filteredTracks = 0,
+    useGlobalTracks,
+    usePvContributor,
+    itsClustersCut,
+    hasTpcSignal,
+    tpcClustersCut,
+    tpcCrossedRowsCut,
+    numPions,
+    numKaons,
+    numProtons
+  };
```

Comment:
I would suggest using a `k` prefixing this names (followed by the corresponding letter but capitalized) because they are constant. The code would be more readable and less error prone.
Same for other `enum`s